### PR TITLE
Embed and filefeed new config

### PR DESCRIPTION
**Pull request by gaelyn**

### CODE REVIEW

File: pages/api/v1/sync-space.ts [hunk 1]
```diff
@@ -114,18 +114,17 @@ export default async function handler(
...
+      workflow: WorkflowType.FileFeed,
+      userId: user.id,
+      organizationId: user.organizationId,
+      spaceType: space.type,
+      actionType: ActionType.SyncFilefeedRecords,
+    });
 
     res.status(200).json({
       success: true,
-      successfullySyncedFlatfileRecordIds,
```
Comment (johnmosesman) [inline pages/api/v1/sync-space.ts:128]: For the old config this should stay the same value

File: lib/flatfile.ts [hunk 2]
```diff
@@ -45,13 +45,15 @@ const flatfileClient = (workflow: WorkflowType) => {
...
+        process.env.FLATFILE_API_KEY || process.env.ONBOARDING_FLATFILE_API_KEY;
       break;
     case WorkflowType.FileFeed:
-      token = process.env.FILEFEED_API_KEY;
+      token = process.env.FLATFILE_API_KEY || process.env.FILEFEED_API_KEY;
       break;
     case WorkflowType.Embedded:
-      token = process.env.EMBEDDED_FLATFILE_API_KEY;
+      token =
+        process.env.FLATFILE_API_KEY || process.env.EMBEDDED_FLATFILE_API_KEY;
```
Comment (johnmosesman) [inline lib/flatfile.ts:56]: Once we have these set in heroku we can just reference them directly, don't need the ||

**Review by johnmosesman — APPROVED** [review]
one comment otherwise +1

File: lib/flatfile.ts (hunk 2 above)
Comment (gaelyn) [reply lib/flatfile.ts:56]: ok

**Review by johnmosesman — APPROVED** [review]

**Pull request merged**